### DISCUSSION TITLE
start simple server using different IP address

### DISCUSSION THREAD

**Discussion by bradelvy** [bluenviron/mediamtx#1298] [Questions]
I have a server running on IP 10.1.1.16 and would like to run a second on 10.1.1.17.
I have set up the NIC to have both IPs.

Is there a parameter I can give in the "sudo docker run --rm -it --network=host aler9/rtsp-simple-server" command that would allow me to define a different ip?

**Comment by aler9** [comment]
Hello, disable the UDP transport protocol, remove the --network=host flag and expose the first server on 10.1.1.16:

```
sudo docker run --rm -it -e RTSP_PROTOCOLS=tcp -p 10.1.1.16:8554:8554 aler9/rtsp-simple-server
```

Then you can create a second server that is exposed on 10.1.1.17:

```
sudo docker run --rm -it -e RTSP_PROTOCOLS=tcp -p 10.1.1.17:8554:8554 aler9/rtsp-simple-server
```

[a comment by an automated account is not shown]
